source/features/parse-backticks.tsx

```diff
@@ -24,7 +24,9 @@ function init(): void {
 		'.Box-row.js-navigation-item a.link-gray-dark', // `isGlobalDiscussionList`
 		'[data-channel^="check_suites"] a', // `isActions`
 		'.repository-content .pr-toolbar h2', // `isActions` run
-		'#wiki-wrapper h1' // `isWiki`
+		'#wiki-wrapper h1', // `isWiki`
+		'a[data-hydro-click*="issue_cross_references.click"]', // Linked pull requests
```

Comment:
Where does this appear? It's best to include the `is*` as well

Reply:
It's the new linked pull request from github. I will get you a picture in a few hours when I'm at a computer.

Reply:
Do we have a pageDetect for it? Include that in the comment.

Reply:
Its is issue

Reply:
![image](https://user-images.githubusercontent.com/16872793/79864919-4692ed80-83a8-11ea-8a0e-6aafc29c35e8.png)

Reply:
> It's best to include the `is*` as well



> Include that in the line comment.



> It's best to include the `is*` as well



> Include that in the line comment.

Reply:
Sorry I was just jotting it down. I was working on the other PR